docs/sdk/auth/fragments/android/working-with-api.md

```diff
@@ -118,7 +118,27 @@ AWSMobileClient.getInstance().signIn(username, password, null, new Callback<Sign
 });
 ```
 
-## Confirm SignIn
+### Confirm SignIn (MFA)
+
+In order to setup multi-factor authentication, choose `Manual configuration` while setting up auth in the CLI and when you get to the `Multifactor authentication` step, choose these values:
+
+```bash
```

Comment:
Can you set this to `console`? `console` both is semantically more correct as this isn't input for the bash interpreter, and I made a change to suppress the copy button for this particular language. See https://github.com/aws-amplify/docs/pull/1557.